tortoise/backends/mysql/schema_generator.py

```diff
@@ -17,9 +20,8 @@ def __init__(self, *args, **kwargs) -> None:
         )
 
         self.FIELD_TYPE_MAP.update({
-            fields.FloatField: 'DOUBLE',
-            fields.DatetimeField: 'DATETIME(6)',
-            fields.TextField: 'TEXT'
+            fields.FloatField: 'DOUBLE', fields.DatetimeField: 'DATETIME(6)', fields.TextField:
+            'TEXT'
```

Comment:
@grigi I turned on yapf in my text editor and this is how it reformatted the code. Looks strange to me, but I haven't used yapf yet so I don't know if this is normal?

Reply:
We tried code formatters, both black and yapf, and was unsatisfied with either of them.
We decided it was better to stick with human formatted code, as it was much more readable.
Generally we are not too strict re style, flake8 checks for the worst offenders, and then it should be readable. 😐